packages/remotemetrics/events.go

```diff
@@ -56,6 +62,29 @@ type MessageFinalizedMetrics struct {
 	LikeEdgeCount      int       `json:"likeEdgeCount,omitempty" bson:"likeEdgeCount"`
 }
 
+// MessageScheduledMetrics defines the scheduling message confirmation metrics record that is sent to remote logger
+type MessageScheduledMetrics struct {
+	Type               string    `json:"type" bson:"type"`
+	NodeID             string    `json:"nodeID" bson:"nodeID"`
+	MetricsLevel       uint8     `json:"metricsLevel" bson:"metricsLevel"`
+	MessageID          string    `json:"messageID" bson:"messageID"`
+	IssuedTimestamp    time.Time `json:"issuedTimestamp" bson:"issuedTimestamp"`
+	SolidTimestamp     time.Time `json:"solidTimestamp,omitempty" bson:"solidTimestamp"`
+	ScheduledTimestamp time.Time `json:"scheduledTimestamp,omitempty" bson:"scheduledTimestamp"`
+	BookedTimestamp    time.Time `json:"bookedTimestamp" bson:"bookedTimestamp"`
+	QueuedTimestamp    time.Time `json:"queuedTimestamp" bson:"queuedTimestamp"`
+	DroppedTimestamp   time.Time `json:"droppedTimestamp,omitempty" bson:"DroppedTimestamp"`
+	AccessMana         float64   `json:"accessMana" bson:"accessMana"`
+}
+
+// MessageSolidificationMetrics defines message solidification record that is sent to the remote logger
+type MissingMessageMetrics struct {
+	Type         string `json:"type" bson:"type"`
+	NodeID       string `json:"nodeID" bson:"nodeID"`
+	MetricsLevel uint8  `json:"metricsLevel" bson:"metricsLevel"`
+	MessageID    string `json:"messageID" bson:"messageID"`
```

Comment:
Should we also add issuer ID here? To analyze how many messages are missing from a given node? What do you think?

Reply:
done